config/routes.rb

```diff
@@ -9,5 +9,14 @@
   end
   get '/home-developers', to: 'pages#home_developers', as: :home_developers # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :dashboards, only: [:index]
+
+  resources :orders, only: [:show, :create] do
+     resources :payments, only: [:new]
+   end
+  resources :investments, only: [] do
+    resources :orders, only: [:new]
```

Comment:
why is investments empty here?

Reply:
mmh that's actually a good question i'm not sur sorry, We can ask claire , this comes from a ticket with her